qemu_v8.mk

```diff
@@ -40,27 +37,37 @@ include toolchain.mk
 ARM_TF_EXPORTS ?= \
 	CROSS_COMPILE="$(CCACHE)$(AARCH64_CROSS_COMPILE)"
 
+ARM_TF_DEBUG ?= $(DEBUG)
+ifeq ($(ARM_TF_DEBUG),0)
+ARM_TF_LOGLVL ?= 30
+ARM_TF_OUT = $(ARM_TF_PATH)/build/qemu/release
+else
+ARM_TF_LOGLVL ?= 50
+ARM_TF_OUT = $(ARM_TF_PATH)/build/qemu/debug
+endif
+
 ARM_TF_FLAGS ?= \
 	BL32=$(OPTEE_OS_HEADER_V2_BIN) \
 	BL32_EXTRA1=$(OPTEE_OS_PAGER_V2_BIN) \
 	BL32_EXTRA2=$(OPTEE_OS_PAGEABLE_V2_BIN) \
 	BL33=$(EDK2_BIN) \
-	ARM_TSP_RAM_LOCATION=tdram \
 	PLAT=qemu \
-	DEBUG=0 \
-	LOG_LEVEL=50 \
+	ARM_TSP_RAM_LOCATION=tdram \
 	BL32_RAM_LOCATION=tdram \
-	SPD=opteed
+	SPD=opteed \
+	DEBUG=0 \
+	LOG_LEVEL=50
```

Comment:
`$(ARM_TF_DEBUG)` and `$(ARM_TF_LOGLVL)`

Reply:
Yes, those flags should be used.